tensorflow/core/distributed_runtime/master.cc

```diff
@@ -526,20 +514,7 @@ void Master::ListDevices(const ListDevicesRequest* req,
                          ListDevicesResponse* resp, MyClosure done) {
   SchedClosure([this, req, resp, done]() {
     if (!req->session_handle().empty()) {
-      MasterSession* session = nullptr;
-      {
-        mutex_lock l(mu_);
-        session = gtl::FindPtrOrNull(sessions_, req->session_handle());
-        if (session != nullptr) {
-          session->Ref();
-        }
-      }
-      if (session == nullptr) {
-        done(errors::InvalidArgument(
-            "Session ", req->session_handle(),
-            " is not found. Possibly, this master has restarted."));
```

Comment:
This change would remove the informative "Possibly, this master has restarted." from the error message. It might be better to have `FindMasterSession()` return `nullptr` in the event that the session isn't found, and allow the caller to send the appropriate error response.